flexmeasures/app.py

```diff
@@ -22,6 +21,10 @@
 
 from flexmeasures.data.services.job_cache import JobCache
 
+# This env variable prevents Flask from loading .env automatically,
+# as we do that manually and early in flexmeasures.utils.config_utils.py
```

Comment:
Where do we do that manually now? I believe now we are sabotizing `read_env_vars()` with this - let's not prevent it and remove this.